news/check-abcs.rst

```diff
@@ -0,0 +1,3 @@
+**Fixed:**
+
+- Linter: handle new versions of `ruamel.yaml` appropriately instead of complaining about `expected to be a dictionary, but got a CommentedMap`. (#871)
```

Comment:
@scopatz will this news format work?

Reply:
Whoops, sorry, will make it a `*`

Reply:
Actually I meant more that all the other sections have been removed, we usually just leave them blank

Reply:
Oh, really? I thought I was supposed to remove them...misremembering clearly. I can add the other sections back in I guess.

Reply:
https://github.com/conda-forge/conda-smithy/blob/51290718737e248a5270ea5f8a819509e3958155/news/rerender.rst worked, presumably, so I think it's okay.

Reply:
great!